### DISCUSSION TITLE
chatgpt is not able to access and read files

### DISCUSSION THREAD

**Discussion by christianpeirson** [llm-workflow-engine/llm-workflow-engine#69] [Q&A]
I have tried to reinstall the program

and

I have tried to use the following commands with no success:

chatgpt !file /Users/Shared/sampletext.txt
!file /Users/Shared/sampletext.txt
file /Users/Shared/sampletext.txt

This is the output:

I'm sorry, I am a language model and do not have the capability to access files on your computer. Is there something specific you would like me to help you with regarding the file or text that it contains?

I do not understand the problem because last week chatgpt read the file absolutely fine. 

What is causing the issue?

**Comment by douglasdnn** [comment]
Looks like you are sending the commands as a normal string to the prompt. @mmabrouk made some exception words (!read, !log, etc) to inform the script that that specific imput is not an ordinary chat promp, but it works from the shell, not (yet) from a python script.
